### PR TITLE
mixin: Add SR to remote_write dashboard

### PR DESCRIPTION
**Pull request by thampiotr**
#### PR Description

- Adds SR to remote_write section on mixin dashboard
- Fixes some issues with grizzly locally

#### Notes to the Reviewer

image

#### PR Checklist

- [x] CHANGELOG.md updated
- [ ] Documentation added
- [ ] Tests updated
- [ ] Config converters updated

### CODE REVIEW

File: example/images/grizzly/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,3 @@
 FROM golang:1.22-alpine
 
-# NOTE(rfratto): Versions of grr newer than v0.2.1 don't work with our Grizzly
-# jsonnet files, reporting that the the Alloy dashboard folder "cannot be
-# found" after creating it.
-RUN go install github.com/grafana/grizzly/cmd/grr@v0.2.1
+RUN go install github.com/grafana/grizzly/cmd/grr@v0.4.3
```
Comment (thampiotr) [inline example/images/grizzly/Dockerfile:3]: Will need to bump it once we land https://github.com/grafana/grizzly/pull/458 upstream.

File: operations/alloy-mixin/dashboards/alloy-logs.libsonnet [hunk 2]
```diff
@@ -28,9 +29,10 @@ local logsDashboard = import 'github.com/grafana/jsonnet-libs/logs-lib/logs/main
             {
               logs+: g.dashboard.withLinksMixin($.grafanaDashboards['alloy-resources.json'].links)                     
                      + g.dashboard.withRefresh('10s')
-                     + g.dashboard.withTagsMixin($._config.dashboardTag),
+                     + g.dashboard.withTagsMixin($._config.dashboardTag)
+                     + g.dashboard.withUid(std.md5(dashboardName)),
             },
         },
-      'alloy-logs.json': alloyLogs.dashboards.logs,
+      [dashboardName]: alloyLogs.dashboards.logs,
```
Comment (thampiotr) [inline operations/alloy-mixin/dashboards/alloy-logs.libsonnet:36]: Without these changes, `grr` will not be able to push the dashboards.

File: operations/alloy-mixin/dashboards/prometheus.libsonnet [hunk 3]
```diff
@@ -87,6 +87,72 @@ local filename = 'alloy-prometheus-remote-write.json';
...
+                    sum(rate(prometheus_remote_storage_samples_total{%(instanceSelector)s, component_path=~"$component_path", component_id=~"$component", url=~"$url"}[$__rate_interval]))
+                )
+            )
+          ||| % $._config,
+          legendFormat='% of samples successfully sent',
+        ),
+      ])
+    ),
+
+    // Write latency
```
Comment (thampiotr) [inline operations/alloy-mixin/dashboards/prometheus.libsonnet:119]: This is just moved from below. I like SR & latency to be the topline as they're main objectives.

**Review by mattdurham — APPROVED** [review]
Gave a very vague overview, LGTM

**Pull request merged**